src/XMakeTasks/ManifestUtil/SecurityUtil.cs

```diff
@@ -762,20 +762,33 @@ internal static string GetPathToTool(System.Resources.ResourceManager resources)
         {
 #pragma warning disable 618 // Disabling warning on using internal ToolLocationHelper API. At some point we should migrate this.
```

Comment:
@rainersigwald  not sure if this pragma warning is still needed. The methods from **ToolLocationHelper** are public (not internal ). It is still showing a warning as the methods used from **ToolLocationHelper** are marked with Obsolete but I couldn't find the recommended method **GetPlatformSDKLocationFile**, can you share some light on this ?